### ISSUE TITLE
Add License and Credits

### ISSUE THREAD

**Issue by ivan-mogilko** [adventuregamestudio/ags-demo-game#1]
Add LICENSE.md with a license(s) text(s).

Add CREDITS.md with a list of asset authors and other people involved in creating this game.

**Comment by ivan-mogilko** [comment]
Added LICENSE.md and respective license texts in separate files.

Added preliminary CREDITS.md, but still has to add a list of authors for additional materials.

**Comment by ericoporto** [comment]
I don’t see the preliminary CREDITS.md

Edit: I think it is the COPYRIGHT.md ? It’s missing your name there @ivan-mogilko. Should also add @karajokalinowski ?

**Comment by karajokalinowski** [comment]
> I don’t see the preliminary CREDITS.md
> 
> Edit: I think it is the COPYRIGHT.md ? It’s missing your name there [@ivan-mogilko](https://github.com/ivan-mogilko). Should also add [@karajokalinowski](https://github.com/karajokalinowski) ?

It's up to you. You can. I just did some minor fixes based on crimsonwizard posting on the discord that it could use updating. My name is just my name, Kara Jo Kalinowski if you want to.

**Comment by ivan-mogilko** [comment]
> It’s missing your name there @ivan-mogilko 

I am not this game's developer, so there's no need to put my name there.

**Comment by karajokalinowski** [comment]
Yeah me either, I just updated a few things that were suggested on the forum and that I found.

**Comment by ivan-mogilko** [comment]
> Yeah me either, I just updated a few things that were suggested on the forum and that I found.

That's a reason to add you to the list of contributors. We add everyone who contributed at least a minor thing.

**Comment by ivan-mogilko** [comment]
Actually, this issue seems to be solved already, so I will close it.
